spec/game_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../classes/game'
+require 'time'
+
+describe Game do
+  context 'When testing the Game class' do
+    game = Game.new(20, '2020-10-02', 'Y', '2020-02-10')
+
+    it 'Should check if game is multiplayer Y' do
+      expect(game.multiplayer).to eq 'Y'
+    end
+
+    it 'should detect the date the album was published' do
```

Comment:
Kindly make sure that ALL public `methods` in all classes have unit tests.

- The `can_be_archived?` and `move_to_archive` methods in the `Item` class should have unit tests.
- The `can_be_archived?` method in the `Game` class should have unit tests.
- The `can_be_archived?` method in the `MusicAlbum` class should have unit tests. Athough you did test if `on_spotify` retuns true/false, the test should use the method `can_be_archived?` to test if music album is `on_spotify`.
- Same thing with the `Book` class. Athought you did test the `cover_state`, kinldy test the state of the cover using the `can_be_archived?` method.